soccer/src/soccer/strategy/agent/position/zoner.hpp

```diff
@@ -30,8 +30,24 @@ class Zoner : public Position {
     ~Zoner() override = default;
     Zoner(const Position& other);
     Zoner(Zoner&& other) = default;
-    Zoner& operator=(const Zoner& other) = default;
-    Zoner& operator=(Zoner&& other) = default;
+    // Zoner& operator=(const Zoner& other) = default;
```

Comment:
you're actually right here, this is pretty frustrating; it's why `const` members are very discouraged. however, i'm scared to write these explicitly because we'd have to maintain them if we ever add members. i would say it's better to either explicitly `= delete` these operators (we rarely would have a use for them anyway) or simply remove the `const` qualifiaction from the member (i don't even see it, but i have a hunch it could be robot_id in the super class)